src/formatDate.js

```diff
@@ -50,7 +50,62 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const inputSeparator = fromFormat[3];
+  const outputSeparator = toFormat[3];
+  const formatDay = 'DD';
+  const formatMonth = 'MM';
+  const shortFormatYear = 'YY';
+  const maxYear = 30;
+  const nineteenthСentury = '19';
+  const twentiethCentury = '20';
```

Comment:
I would change the names of these variables.If the year is 1950, it is not the nineteenth century